src/styles/base/_override.links.scss

```diff
@@ -1,49 +1,17 @@
-%link-inverse {
-  color: $color-white;
-
-  &:visited {
-    color: $color-white;
-  }
-
-  &:hover,
-  &:active {
-    color: hsla(0, 0%, 100%, 0.77);
-  }
-}
-
-%link,
-%link-darker {
-  color: $color-primary-alt-light;
-
-  &:visited {
-    color: $color-visited;
-  }
-
-  // Override visited with hover
-  &:hover,
-  &:active {
-    color: $color-primary-alt;
-    outline: 0 none;
-  }
-
-  &:focus {
-    box-shadow: 0 0 0 4px $focus-choice-color-highlight,
-      0 0 0 2px $focus-choice-color;
-    border-color: $focus-choice-color;
-    outline: none;
-  }
-
-  &.mct-link--inverse {
```

Comment:
`.mct-link--inverse` is being used in 2 places on coverage tools - https://github.cms.gov/search?q=mct-link--inverse&type=code

Could `.ds-c-link--inverse` be used instead - why was `.mct-link--inverse` needed?

Reply:
@babsmiles Maybe you know more about this?

Reply:
Hmm, yeah. I'm not sure how this got included at a design system level. This "mct" is seen in the MCT repo. When replacing the link with the .ds-c-link--inverse class the styling doesn't change. I think we could remove this class.

Reply:
Thanks for checking on this @babsmiles , can you reach out to the mct team to let them know to make that update?

Reply:
Here is a ticket and am discussing with the MCT team about next steps. https://jira.cms.gov/browse/WNMGDS-990